bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/TargetFeedback.scala

```diff
@@ -194,6 +195,7 @@ object BagsFeedback {
   }
 
   case object NoStarsRow      extends BagsStateRow(BANANA.some, "Automatic guide star lookup found nothing.", warningIcon)
+  case object NoGuideStateRow extends BagsStateRow(BANANA.some, "Observation unguided.", warningIcon)
```

Comment:
Just a small nit to pick about the name of this `case object`.  The other *`StateRow`s correspond to actual BAGS manager states but this one doesn't.  Maybe a more appropriate name might be `AgsOffRow`.

What happens if you have manual guide stars?  Do you still see "Observation unguided."?  That would be misleading and we would need to at least change the message to something like "Auto guide star search disabled." if so.

Reply:
> Just a small nit to pick about the name of this case object. The other *StateRows correspond to actual BAGS manager states but this one doesn't. Maybe a more appropriate name might be `AgsOffRow`.

I didn't know. I'll change it to `AgsOffRow`.

> What happens if you have manual guide stars? Do you still see "Observation unguided."? That would be misleading and we would need to at least change the message to something like "Auto guide star search disabled."

The message is shown only when selecting "base" or the previously created guide star. It's not shown when clicking on the rows "Auto" or "Manual". It's a bit weird, I may have to record another screen capture.

Reply:
> I'll change the message to "Auto guide star search disabled", but there is still something weird going on. The message is shown only when selecting "base" or the previously created guide star. It's not shown when clicking on the rows "Auto" or "Manual". I may have to record another screen capture.

I think that may be correct.  What still concerns me though is that the message appears when auto guide search is off and yet there are manual guide stars?  It seems like it should only be shown if there isn't a selected manual guide star group.